src/MCPServer/lib/unit.py

```diff
@@ -44,8 +44,8 @@ def reloadFileList(self):
         self.fileList = {}
         # currentPath must be a string to return all filenames as bytestrings,
         # and to safely concatenate with other bytestrings
-        currentPath = str(self.currentPath.replace("%sharedPath%", \
-                                                       archivematicaMCP.config.get('MCPServer', "sharedDirectory"), 1) + "/")
+        currentPath = (self.currentPath.replace("%sharedPath%", \
+                                                       archivematicaMCP.config.get('MCPServer', "sharedDirectory"), 1) + "/").encode('utf-8')
```

Comment:
Nitpick: Having both () and \ to handle the newline is overkill, I'd probably just put it all on one line and let the editor handle wrapping (especially since it's only 9 characters shorter split)